### ISSUE TITLE
create login page (HTML structure)

### ISSUE THREAD

**Issue by matei1337** [matei1337/domain-registry-app#9]

**Comment by Trevy05** [comment]
Is this still available? I'd like to work on it.

**Comment by matei1337** [comment]
> Is this still available? I'd like to work on it.

Yes, it is still available. You can start working on it, but please create a new branch from this issue.

After you finish working on it, create a PR from your branch to `develop` branch, thank you!

By the way, this project is created by using Angular for frontend.

Also, you can add me on Discord: danuts#4874 if you want to collaborate with me on this project.
